pkg/kudoctl/cmd/verify/verify.go

```diff
@@ -13,37 +13,43 @@ type ParamWarnings []ParamWarning
 type ParamError string
 type ParamErrors []ParamError
 
-var verifiers = []ParameterVerifier{
+var verifiers = []PackageVerifier{
 	DuplicateVerifier{},
 	InvalidCharVerifier{";,"},
+	TemplateParametersVerifier{},
+	TemplateVerifier{},
```

Comment:
Maybe rather `TemplateReferenceVerifier`? I was wondering what it does before I looked up the comment below